Pipfile

```diff
@@ -0,0 +1,142 @@
+[[source]]
+
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+
+[packages]
+
+alabaster = "==0.7.10"
+"asn1crypto" = "==0.24.0"
```

Comment:
@UmanShahzad This is the first time I'm learning about `pipfile`. Why wrap some package names with quotes but not all of them?

Reply:
@itsjeyd It turns out `pipenv install` generates a `Pipfile` for you if it doesn't exist but a `requirements.txt` does (which is probably the main way they'll get masses to switch to it eventually), so this was generated by it. The pattern shows that quotes are around any name with a number in it, like asn**1**crypto, etc..